host-interaction/process/create/execute-command-through-syscall.yml

```diff
@@ -0,0 +1,12 @@
+rule:
+  meta:
+    name: execute command through syscall
+    namespace: host-interaction/process/create
+    author: alyssa.rahman@fireeye.com
+    scope: file
+    examples:
+      - 3db3e55b16a7b1b1afb970d5e77c5d98
+  features:
+    - and:
+      - os: linux
+      - import: syscall
```

Comment:
`syscall` is an API that directly invokes the system call provided as an argument (i think, as documented here: https://man7.org/linux/man-pages/man2/syscall.2.html). so, this rule needs more specificity to disambiguate execution of a command from the other system calls (like recv, write, etc.).